test/README.md

```diff
@@ -63,18 +68,34 @@ output with given format JSON by using *structs* defined in inspect package.
 You can run the entire suite of integration tests with the following command:
 
 ```
-GOPATH=~/go ginkgo -tags "remote" -v test/e2e/.
+make localintegration
+```
+
+To run the remote tests use:
+```
+make remoteintegration
 ```
 
-Note the trailing period on the command above. Also, **-v** invokes verbose mode.  That
-switch is optional.
+### Test variables
 
+Some test only work as rootless while others only work as root. So to test everything
+you should make sure to run the make command above as normal user and root.
+
+The following environment variables are supported by the test setup:
+ - `PODMAN_BINARY`: path to the podman binary, defaults to `bin/podman` in the repository root.
+ - `PODMAN_REMOTE_BINARY`: path to the podman-remote binary, defaults to `bin/podman-remote` in the repository root.
+ - `QUADLET_BINARY`: path to the quadlet binary, defaults to `bin/quadlet` in the repository root.
+ - `CONMON_BINARY`: path to th conmon binary, defaults to `/usr/libexec/podman/conmon`.
+ - `OCI_RUNTIME`: which oci runtime to use, defaults to `crun`.
+ - `NETWORK_BACKEND`: the network backend, either `cni` (default) or `netavark`.
+ - `PODMAN_DB`: the database backend `boltdb` (default) or `sqlite`.
+ - `PODMAN_TEST_IMAGE_CACHE_DIR`: path were the container images should be cached, defaults to `/tmp`.
```

Comment:
Thanks so much!  I desired this in the docs for a long time but never sat down to do it myself.